docs/source/markdown/podman-create.1.md

```diff
@@ -678,7 +678,8 @@ Valid _mode_ values are:
   - **outbound_addr6=INTERFACE**: Specify the outbound interface slirp should bind to (ipv6 traffic only).
   - **outbound_addr6=IPv6**: Specify the outbound ipv6 address slirp should bind to.
   - **port_handler=rootlesskit**: Use rootlesskit for port forwarding. Default.
-  - **port_handler=slirp4netns**: Use the slirp4netns port forwarding.
+  Note: Rootlesskit changes the source ip of incoming packets to a IP in the container network namespace, usually `10.0.2.100`. If your application requires the real source ip, e.g. web server logs, use the slirp4netns port handler. This port handler is also used for rootless users when connected to user-defined networks.
```

Comment:
> This port handler is also used for rootless users when connected to user-defined networks.

Perhaps "rootless users" -> "rootless containers"

Reply:
Also, "This port handler" -> "RootlessKit port handler" might be better. (Some people may interpret "this" to be slirp4netns that is mentioned in the previous sentence)

Reply:
nit: "ip" -> "IP address"